coral-component-list/src/scripts/SelectList.js

```diff
@@ -402,9 +430,14 @@ class SelectList extends BaseComponent(HTMLElement) {
    
    @private
    */
-  _resetTabTarget() {
-    const selectedItems = this.items._getAllSelected().filter(item => !item.hasAttribute('hidden'));
-    this._tabTarget = selectedItems.length ? selectedItems[0] : this.items._getSelectableItems().filter(item => !item.hasAttribute('hidden'))[0];
+  _resetTabTarget(forceFirst = false) {
+    let items = this.items._getAllSelected().filter(item => !item.hasAttribute('hidden'));
+
+    if (items.length === 0) {
+      items = this.items._getSelectableItems().filter(item => !item.hasAttribute('hidden'));
+    }
+    
+    this._tabTarget = forceFirst ? items[0] : (items.find(item => item.tabIndex === 0) || items[0]); 
```

Comment:
function at L-270:
  _onBlur() {
    // required otherwise the latest item that had the focus would get it again instead of the selected item
    this._resetTabTarget();
  }
Does this change breaks it by setting tabTarget to the last focused item?

Reply:
At https://github.com/adobe/coral-spectrum/pull/127/files#diff-6038a55acfa892578138ba3bf357f8d9255a95ad38761f20cc8fc2e4745fd0f1R434, we are using `this.items._getAllSelected()`, so we prioritize the selected items over selectable items.